index.html

```diff
@@ -59,6 +59,216 @@ <h3>LET'S CONNECT</h3>
         </ul>
       </section>
     </div>
+    <!-- ProjectsSection -->
+    <div class="projectsContainer">
+        <div class="project1">
+          <div class="projImg">
+            <img src="./Images/mobileprojects/project1Img.svg" alt="project1img" />
+          </div>
+          <h3>Tonic</h3>
+          <ul class="a">
+            <li>CANOPY</li>
+            <li>
+              <img src="./Images/icons/CounterDot.svg" alt="counterimg" />
+            </li>
+            <li>Back EndDev</li>
+            <li>
+              <img src="./Images/icons/CounterDot.svg" alt="counterimg" />
+            </li>
+            <li>2015</li>
+          </ul>
+          <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="b">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+          </ul>
+          <button>
+              See project
+          </button>
```

Comment:
- [ ] Great job on creating buttons for all of your portfolio items :+1:. However, it's important to note that as a project requirement, the buttons should follow the style of Active, Hover, and Pressed. Please ensure that the buttons are designed according to these specifications.

![Screenshot from 2023-03-23 05-31-18](https://user-images.githubusercontent.com/105117832/227176576-64d42f3b-41b1-47ea-8faf-04e216f23ebe.png)